subprojects/wrapper/src/integTest/groovy/org/gradle/integtests/WrapperLoggingIntegrationTest.groovy

```diff
@@ -17,6 +17,11 @@
 package org.gradle.integtests
 
 class WrapperLoggingIntegrationTest extends AbstractWrapperIntegrationSpec {
+
+    def setup() {
+        executer.withWelcomeMessageEnabled()
```

Comment:
Does this need an isolated user home to prevent a future integration test from conflicting with it that also enables the welcome message?

Reply:
The `wrapperExecutor` already does that: https://github.com/gradle/gradle/blob/master/subprojects/wrapper/src/integTest/groovy/org/gradle/integtests/AbstractWrapperIntegrationSpec.groovy#L38

Reply:
Ah, good. I didn't realize this was a special kind of executer.